pkg/infra/fs/copy.go

```diff
@@ -157,12 +157,6 @@ func CopyRecursive(src, dst string) error {
 				return err
 			}
 		}
-
-		if srcFi.Mode()&os.ModeSymlink != 0 {
-			if err := os.Chmod(dstPath, srcFi.Mode()); err != nil {
-				return err
-			}
```

Comment:
this never ran as far as i can tell because `srcFi.Mode` was the output of `os.Stat`. it seems like we wouldn't want to run chmod in the case of a symlink. i might be wrong, but i think it would modify the source file and not the link itself